pkg/model/stack.go

```diff
@@ -50,11 +50,11 @@ type Stack struct {
 	Volumes   map[string]*VolumeSpec `yaml:"volumes,omitempty"`
 	Namespace string                 `yaml:"namespace,omitempty"`
 	Context   string                 `yaml:"context,omitempty"`
-	Services  composeServices        `yaml:"services,omitempty"`
+	Services  ComposeServices        `yaml:"services,omitempty"`
```

Comment:
Nit: it is a bit odd that we need to make this public just for a test. How is this created in other places? as a result of marshaling/unmarshaling?

Reply:
All other tests referring to services are declared in `model` package. We could be using `map[string]*model.Service` instead but I think we should be using the defined struct in all other tests too...